README.md

```diff
@@ -14,18 +14,63 @@ All bugs, stories, and backlog for this project are managed through [Hyperledger
 in project IS (note that regular Indy tickets are in the INDY project instead...). Also, join
 us on [Hyperledger's Rocket.Chat](https://chat.hyperledger.org/) at #indy-sdk to discuss.
 
-## Building Indy SDK
+## How-to-install
+### Instal for Ubuntu based distro (Ubuntu 16.04)
+It is recommended to install packages with APT (change stable to `master` or `rc` if needed):
```

Comment:
sudo add-apt-repository "deb https://repo.sovrin.org/sdk/deb xenial {release-channel}"

{release-channel} can be:

master - for master release channel
rc - for rc release channel
stable - for stable release channel

Reply:
Please spell "install" with 2 Ls